### PR TITLE
Force next@12.2.4 until esbuild update

### PR DESCRIPTION
**Pull request by dcousens**
This pull request needs to merged in the interim until the following two issues are resolved:

- https://github.com/keystonejs/keystone/issues/7902
- https://github.com/keystonejs/keystone/issues/7816

After they are resolved, we can revert these changes.
The changes in this pull requests should be published immediately after merging.

### CODE REVIEW

**Review by flexdinesh — APPROVED** [review]

File: create-keystone-app/starter/package.json [hunk 1]
```diff
@@ -13,9 +13,15 @@
     "@keystone-6/core": "^2.1.0",
     "@keystone-6/fields-document": "^4.0.1",
     "graphql": "^15.8.0",
+    "next": "12.2.4",
     "typescript": "^4.7.4"
   },
-  "engines": {
-    "node": "^14.15 || ^16.13"
+  "overrides": {
```
Comment (dcousens) [inline create-keystone-app/starter/package.json:19]:
```suggestion
  "// yarn": "this is a temporary workaround for yarn users"
  "overrides": {
```

File: create-keystone-app/starter/package.json [hunk 2]
```diff
@@ -13,9 +13,15 @@
...
+    "graphql": "^15.8.0",
+    "next": "12.2.4"
+  },
```
Comment (dcousens) [inline create-keystone-app/starter/package.json:22]:
```suggestion
  },
  "// npm": "this is a temporary workaround for npm users"
```

**Review by dcousens — COMMENTED** [review]
Comments might be helpful for those unaware of why these exist

**Comment by dcousens** [comment]
This was not how it should have been, please see https://github.com/keystonejs/create-keystone-app/commit/3b4360a114f00094e40fdc89dd4c82e1456b9ae5 for the fixed diff.

**Pull request merged**